CMakeLists.txt

```diff
@@ -31,4 +31,17 @@ set(PROFILER_SOURCES
     ${CMAKE_CURRENT_LIST_DIR}/src/external/runtime/src/coreclr/pal/prebuilt/idl/corprof_i.cpp
     )
 
+# Add version files
+if(CLR_CMAKE_HOST_WIN32)
+    set(SOURCES
+        ${SOURCES}
+        ${PROJECT_BINARY_DIR}/NativeVersion.rc
+        )
+else(CLR_CMAKE_HOST_WIN32)
```

Comment:
Without having much familiarity with this format, is there a reason why we have a condition for an `else`?

Reply:
It's the same condition in the above `if` but it's not really a condition. I believe older versions of CMake required this pairing and in some newer versions it issues a warning if it doesn't have the repeated condition.

Reply:
See https://cmake.org/cmake/help/latest/command/if.html